README.md

```diff
@@ -749,25 +749,26 @@ imgGroup = L.distortableCollection({
 
 There are [plenty of outstanding issues to resolve](https://github.com/publiclab/Leaflet.DistortableImage/issues). Please consider helping out!
 
-1. This project uses `grunt` to do a lot of things, including concatenate source files from `/src/` to `/DistortableImageOverlay.js`:
+1. This project uses `Webpack` to bundle JavaScript.
 
 ```Bash
 $ npm install
 
 # install leaflet peer dependency locally
 $ npm install leaflet --no-save
 
-# It'll watch for changes and concatenate them on the fly
-$ grunt
+# It'll open examples/index.html in your default browser and watch live for changes (Webpack development server compiles the changes and refreshes the browser)
+$ npm run dev
 ```
 
-2. To build all files from `/src/` into the `/dist/` folder, run:
+2. Once you finish, build all files from `/src/` into the `/dist/` folder:
 
 ```Bash
-$ grunt concat:dist
+$ npm run build
```

Comment:
While we're waiting for Sasha to review, shall we add this to our travis.yml file? That way, any changes to upstream dependencies required to actually run webpack will also be checked for by Travis. Thanks!

Reply:
Yes, let's add it :tada: